src/test/kotlin/org/jitsi/nlj/rtp/codec/vp9/Vp9PacketTest.kt

```diff
@@ -0,0 +1,708 @@
+            descriptorSize = descriptorSize,
+            scalabilityStructure = scalabilityStructure
+        )
+    }
+
+    /* Packets captured from Chrome VP9 call */
+    private val testPackets = arrayOf(
+        /* Live video - Chrome 81 */
+        SampleVp9Packet(
+            // RTP
+        "906536b69f3077686098017b" +
+                // RTP header extension
+                "bede0002" + "3202168751210700" +
+                // I=1,P=0,L=1,F=0,B=1,E=0,V=1,Z=0
+                "aa" +
+                // M=1,PID=0x1e65=7781
+                "9e65" +
+                // TID=0,U=0,SID=0,D=0
+                "00" +
+                // TL0PICIDX=0xfd=253
+                "fd" +
+                // Begin SS: N_S=2,Y=1,G=1
+                "58" +
+                // WIDTH=320
+                "0140" +
+                // HEIGHT=180
+                "00b4" +
+                // WIDTH=640
+                "0280" +
+                // HEIGHT=360
+                "0168" +
+                // WIDTH=1280
+                "0500" +
+                // HEIGHT=720
+                "02d0" +
+                // N_G=4
+                "04" +
+                // TID=0,U=0,R=1
+                "04" +
+                // P_DIFF=4
+                "04" +
+                // TID=2,U=1,R=1
+                "54" +
+                // P_DIFF=1
+                "01" +
+                // TID=1,U=1,R=1
+                "34" +
+                // P_DIFF=2
+                "02" +
+                // TID=2,U=1,R=1
+                "54" +
+                // P_DIFF=1
+                "01" +
+                // VP9 media.  Truncated.
+                "834983420013f00b3827f8167858e0907063a8000f",
+            isStartOfFrame = true,
+            isEndOfFrame = false,
+            isEndOfPicture = false,
+            isKeyframe = true,
+            isInterPicturePredicted = false,
+            pictureId = 7781,
+            hasExtendedPictureId = true,
+            isUpperLevelReference = true,
+            tid = 0,
+            sid = 0,
+            isSwitchingUpPoint = false,
+            usesInterLayerDependency = false,
+            tL0PICIDX = 253,
+            descriptorSize = 27,
+            scalabilityStructure = RtpEncodingDesc(0x6098017bL,
+                arrayOf(
+                    RtpLayerDesc(0, 0, 0, 180, 7.5, null),
+                    RtpLayerDesc(0, 1, 0, 180, 15.0, null /* TODO */),
+                    RtpLayerDesc(0, 2, 0, 180, 30.0, null /* TODO */),
+                    RtpLayerDesc(0, 0, 1, 360, 7.5, null /* TODO */),
+                    RtpLayerDesc(0, 1, 1, 360, 15.0, null /* TODO */),
+                    RtpLayerDesc(0, 2, 1, 360, 30.0, null /* TODO */),
+                    RtpLayerDesc(0, 0, 2, 720, 7.5, null /* TODO */),
+                    RtpLayerDesc(0, 1, 2, 720, 15.0, null /* TODO */),
+                    RtpLayerDesc(0, 2, 2, 720, 30.0, null /* TODO */)
+                    ))
+        ),
+        SampleVp9Packet(
```

Comment:
Are the rest of these packets notable in some way?  It'd be nice to give a summary at the start of each one (even better might be to include a description in the sample packet itself, and then you can include that context in `withClue` wrapper for the test so the failure is less ambiguous)

Reply:
Ah, good idea, I'll do this.